llvm/lib/TableGen/Record.cpp

```diff
@@ -3140,10 +3140,10 @@ Record::getValueAsListOfStrings(StringRef FieldName) const {
     if (const auto *SI = dyn_cast<StringInit>(I))
       Strings.push_back(SI->getValue());
     else
-      PrintFatalError(getLoc(),
-                      Twine("Record `") + getName() + "', field `" + FieldName +
-                          "' exists but does not have a list of strings value: " +
-                          I->getAsString());
+      PrintFatalError(
```

Comment:
Unrelated formatting change?

Reply:
Yeah, leftover from my previous version. Unfortunately, I haven't been able to convince clang-format to leave it alone even after copying the trunk version of the file to this change. Maybe I need to do something else instead of `git clang-format HEAD^`  maybe `git clang-format main`. I'll try